src/status_im/contexts/communities/actions/addresses_for_permissions/events.cljs

```diff
@@ -73,19 +73,25 @@
 
 (defn check-permissions-to-join-for-selection
   [{:keys [db]} [community-id addresses]]
-  (if (empty? addresses)
-    ;; When there are no addresses we can't just check permissions, otherwise
-    ;; status-go will consider all possible addresses and the user will see the
-    ;; incorrect highest permission role.
-    {:db (update db :communities/permissions-checks-for-selection dissoc community-id)}
-    {:db (assoc-in db [:communities/permissions-checks-for-selection community-id :checking?] true)
-     :fx [[:json-rpc/call
-           [{:method     :wakuext_checkPermissionsToJoinCommunity
-             :params     [{:communityId community-id :addresses addresses}]
-             :on-success [:communities/check-permissions-to-join-during-selection-success
-                          community-id]
-             :on-error   [:communities/check-permissions-to-join-during-selection-failure
-                          community-id addresses]}]]]}))
+  (let [request-id (hash addresses)]
```

Comment:
I think we should use either a random number or something else, `hash` of addresses seems a bit risky an a bit overloaded, though it should be fine

Reply:
done, thank you